frontend/src/lib/components/SortableList/sortableListItem.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+
+import { useElementBoundingRect } from "@lib/hooks/useElementBoundingRect";
+import { createPortal } from "@lib/utils/createPortal";
+import { resolveClassNames } from "@lib/utils/resolveClassNames";
+import { DragIndicator } from "@mui/icons-material";
+
+import { HoveredArea, SortableListContext } from "./sortableList";
+import { SortableListDropIndicator } from "./sortableListDropIndicator";
+
+export type SortableListItemProps = {
+    id: string;
+    title: React.ReactNode;
+    initiallyExpanded?: boolean;
+    startAdornment?: React.ReactNode;
+    endAdornment?: React.ReactNode;
+    children: React.ReactNode;
```

Comment:
In the well-log viewer, the sortable items for the individual plots dont have any child content; the children prop should be optional for items that don't intend to expand